eng/pipelines/publish.yml

```diff
@@ -32,6 +32,18 @@ jobs:
         value: ${{ parameters.artifactsDir }}/AssetManifests
 
     steps:
+      - powershell: |
+          $prefix = "refs/heads/"
```

Comment:
Why is this needed? From what I know, Build.SourceBranch uses the full name? i.e -> refs/heads/release/3.0 ?

Reply:
You're right about the var, this describes `Build.SourceBranch` and `Build.SourceBranchName`: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml.

This trimming step is needed because in dotnet/versions, we don't want `refs/heads/release/3.0`, only `release/3.0`.

Note that the manifest still uses `refs/heads/release/3.0` in this PR, if you didn't notice.

Reply:
Yes that is intentional for the manifest